### ISSUE TITLE
Cannot select the lists targeted in campaign's "Marketing" record.

### ISSUE THREAD

**Issue by tjaafar** [salesagility/SuiteCRM#2896]
#### Issue
In the new campaign Wizard (type "Email"), when I configure the "Marketing" section, it doesn't give me an option to select which target lists to be targeted.

The wizard only allows to select the target lists for the campaign (in general), but not the "Marketing" records.

#### Expected Behavior
In the "Marketing" section, it should allow to select "all" or "any of" the target lists that are added to the campaign in general.

#### Actual Behavior
It automatically assumes that all target lists are included in the created Marketing.

#### Possible Fix
Still not fixed.

#### Steps to Reproduce
1.  From Campaigns, run "Create Campaign", select "Email" type.
2. Make sure you add some Test & Default target lists to the campaign.
3. Fill the forms until you reach to the "Marketing" tab.
3. You will not find a way to select which target lists to be targeted in this "Email Marketing".

#### Your Environment

* SuiteCRM Version used: 7.7.9
* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): Chrome
* Environment name and version (e.g. MySQL, PHP 7): 5.6
* Operating System and version (e.g Ubuntu 16.04): Ubuntu 14

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Cannot select the lists targeted in campaign's "Marketing" record.](https://trello.com/c/YtEf09Xy/41-cannot-select-the-lists-targeted-in-campaign-s-marketing-record)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
